curriculum/challenges/english/blocks/workshop-quincys-job-tips/68eab45aceb7c00fd8de4fed.md

```diff
@@ -13,7 +13,7 @@ Below this heading, add a block quotation element with a `cite` attribute with t
 
 # --hints--
 
-Your should have an `h2` element nested inside the second section.
```

Comment:
There is another instance of this:

https://github.com/freeCodeCamp/freeCodeCamp/blob/c07461c08b69cc0a090fa269cd4983999984aca5/curriculum/challenges/english/blocks/workshop-media-catalogue/68b704862dd6559e890c2a86.md#L24

Please help fix that as well.